source/fundamentals/indexes.txt

```diff
@@ -0,0 +1,411 @@
+:manual:`Data Modeling and Indexes </core/data-model-operations/#indexes>`.
+
+Index Types
+-----------
+
+MongoDB provides several different index types to support querying
+your data. The following sections describe the most common index types
+and provide sample code for creating each index type.
+
+.. note::
+  
+  These example uses the ``sample_mflix.movies`` and ``sample_mflix.theaters`` collections
+  from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+  free MongoDB Atlas cluster and load the sample datasets, see :ref:`csharp-quickstart`.
+
+Single Field Indexes
+~~~~~~~~~~~~~~~~~~~~
+
+**Single-field indexes**  are indexes with a reference to a single field within a
+collection's documents. They improve single field query and sort performance, and support
+:manual:`TTL Indexes </core/index-ttl>` that automatically remove documents from a
+collection after a certain amount of time or at a specific clock time.
+
+.. note::
+
+   The ``_id_`` index is an example of a single-field index. This index is automatically
+   created on the ``_id`` field when a new collection is created.
+
+The following example creates an index in ascending order on the ``title`` field within
+the ``sample_mflix.movies`` collection:
+
+.. literalinclude:: /includes/fundamentals/code-examples/Indexes.cs
+   :language: csharp
+   :start-after: begin-single-index
+   :end-before: end-single-index
+   :dedent:
+
+The following is an example of a query that is covered by the index
+created in the preceding code snippet:
+
+.. literalinclude:: /includes/fundamentals/code-examples/Indexes.cs
+   :language: csharp
+   :start-after: begin-single-index-query
+   :end-before: end-single-index-query
+   :dedent:
+
+For more information, see :manual:`Single Field Indexes </core/index-single>` in the Server manual.
+
+Compound Indexes
+~~~~~~~~~~~~~~~~
+
+**Compound indexes**  hold references to multiple fields within a collection's documents,
+improving query and sort performance.
+
+The following example creates a compound index on the ``type`` and ``rated`` fields within
+the ``sample_mflix.movies`` collection:
+
+.. literalinclude:: /includes/fundamentals/code-examples/Indexes.cs
+   :language: csharp
+   :start-after: begin-compound-index
+   :end-before: end-compound-index
+   :dedent:
+
+The following is an example of a query that is covered by the index
+created in the preceding code snippet:
+
+.. literalinclude:: /includes/fundamentals/code-examples/Indexes.cs
+   :language: csharp
+   :start-after: begin-compound-index-query
+   :end-before: end-compound-index-query
+   :dedent:
+
+For more information, see :manual:`Compound Indexes </core/index-compound>` in the Server manual.
+
+Multikey Indexes
+~~~~~~~~~~~~~~~~
+
+**Multikey indexes**  are indexes that improve performance for queries that specify a
+field with an index that contains an array value. You can define a multikey index using
```

Comment:
Although this is the description we use on other pages, I think it's too wordy and hard to understand. I think the description they use on the Server docs is a little better:
```suggestion
**Multikey indexes** collect and sort data from fields containing array values. You can define a multikey index using
```